### DISCUSSION TITLE
RJSF‘s own schema

### DISCUSSION THREAD

**Discussion by chris-schra** [rjsf-team/react-jsonschema-form#2894] [Q&A]
This might be a stupid question but googling „react-json-schema-form json schema“ will obviously return a lot of results 😃

I‘m looking for rjsf‘s own JSON schema (the one including „ui:widget“ property definition, for example) to be able to extend it (eg adding „data:index“) and validate it

help is really appreciated and in case it doesn‘t exist yet, I could add it. Maybe it makes sense to even include it in the repo so UIs can download it

**Comment by epicfaace** [comment]
There's no schema available for the uiSchema, but feel free to add it. For the JSON Schema, you can check out the meta schemas that are published by the JSON schema organization: https://json-schema.org/specification.html#general-purpose-meta-schema
